# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLLibraryEGL.h
# Commit: edfb73b96d5a
# Full Hash: edfb73b96d5ad946bbdd9da2e98d07b1f4565e1f
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2019-06-15 21:44:24
# Regressor Bug: 1559524
# File Overlap Count: 2
# Description:
#   Bug 1559524 - Reduce calls to GLLibraryEGL::Get(). r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D35103
# ==============================================================================

diff -r 0378898329b2 -r edfb73b96d5a gfx/gl/GLLibraryEGL.h
--- a/gfx/gl/GLLibraryEGL.h	Sat Jun 15 05:14:53 2019 +0000
+++ b/gfx/gl/GLLibraryEGL.h	Sat Jun 15 05:21:12 2019 +0000
@@ -357,9 +357,9 @@
 #undef AFTER_CALL
 #undef MOZ_FUNCTION_NAME
 
-  ////
+      ////
 
-  EGLDisplay Display() const {
+      EGLDisplay Display() const {
     MOZ_ASSERT(mInitialized);
     return mEGLDisplay;
   }